Thorium-Sim/thorium-nova
# Add configuration backend and UI for setting the number of cargo containers a ship has.

## Related Issue








Closes #262.
Closes #351.

## How do you know the changes work correctly?






Manual testing:
Configure a ship plugin.
Go to the "Cargo" setting.
Update the fields. Refresh the browser. The values should be retained.

## Screenshots (if appropriate):
<img width="1904" alt="Screen Shot 2022-06-13 at 7 55 02 PM" src="https://user-images.githubusercontent.com/6558157/173465937-7e73d80f-27fa-416d-87f5-5bc7ee6dd4b9.png">

## Types of changes



      change)

## Checklist:
